alt_src/__init__.py

```diff
@@ -0,0 +1 @@
+from alt_src import *
```

Comment:
this will import everything, include imports from `alt_src.py`.

What about importing only what should be public? Or defining an `__all__` in `alt_src.py`.

Reply:
Yeah, I overlooked this. Will define __all__.